<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HexView</class>
 <widget class="QWidget" name="HexView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1041</width>
    <height>709</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="vSplitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="layoutWidget_2">
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
       <item>
        <widget class="QWidget" name="tbContainer" native="true"/>
       </item>
       <item>
        <widget class="QSplitter" name="hSplitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QWidget" name="layoutWidget_3">
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
           <item>
            <widget class="BinaryNavigator" name="binaryNavigator" native="true">
             <property name="minimumSize">
              <size>
               <width>128</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>128</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QHexEdit" name="hexEdit" native="true">
             <property name="contextMenuPolicy">
              <enum>Qt::CustomContextMenu</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="DataTypesView" name="dataView" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="ActionWidget" name="actionWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::South</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="ChartWidget" name="chartWidget">
       <attribute name="title">
        <string>Chart</string>
       </attribute>
      </widget>
      <widget class="SignaturesWidget" name="signaturesWidget">
       <attribute name="title">
        <string>Signatures</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabFormat">
       <attribute name="title">
        <string>Format</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="FormatTreeView" name="tvFormat">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="uniformRowHeights">
           <bool>true</bool>
          </property>
          <property name="allColumnsShowFocus">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="StringsWidget" name="stringsWidget">
       <attribute name="title">
        <string>Strings</string>
       </attribute>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QHexEdit</class>
   <extends>QWidget</extends>
   <header>qhexedit/qhexedit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FormatTreeView</class>
   <extends>QTreeView</extends>
   <header>formattreeview/formattreeview.h</header>
  </customwidget>
  <customwidget>
   <class>DataTypesView</class>
   <extends>QWidget</extends>
   <header>datatypesview/datatypesview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BinaryNavigator</class>
   <extends>QWidget</extends>
   <header>views/hexview/binarynavigator/binarynavigator.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ActionWidget</class>
   <extends>QWidget</extends>
   <header>actionwidget/actionwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ChartWidget</class>
   <extends>QWidget</extends>
   <header>views/hexview/chartwidget/chartwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StringsWidget</class>
   <extends>QWidget</extends>
   <header>views/hexview/stringswidget/stringswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SignaturesWidget</class>
   <extends>QWidget</extends>
   <header>views/hexview/signatureswidget/signatureswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
